common/openstack/adding-an-openstack-infrastructure-provider.adoc

```diff
@@ -0,0 +1,57 @@
+After initial installation and creation of a {product-title} environment, add an OpenStack Infrastructure provider to the appliance. {product-title} supports operating with the OpenStack `admin` tenant.
```

Comment:
Fixed in: https://github.com/ManageIQ/manageiq_docs/pull/184/commits/87068f77677f6a031edab31278c1f5272f15b5e0